docs/descriptors/repository.md

```diff
@@ -5,18 +5,18 @@
 
 ## Linters
 
-| Linter                                 | Configuration key                           | Status                                                                                                                                     |
```

Comment:
Thank you for the contribution, but see the comment at the top of the file. This file is auto-generated.